airgun/entities/redhat_repository.py

```diff
@@ -29,6 +29,14 @@ def read(self, entity_name, category='Available', widget_names=None):
             entity_data['items'] = view.available.items(name=entity_name)[0].read_items()
         return entity_data
 
+    def recommended_repos(self):
```

Comment:
Can we change the name to read_recommended_repos(), As all other methods start with action? what do you think?